apps/rush-lib/src/schemas/command-line.schema.json

```diff
@@ -41,6 +41,11 @@
             "title": "Is Parallelized",
             "description": "Describes whether or not this command can be run in parallel (with multiple instances running on several projects simultaneously). Defaults to 'true'.",
             "type": "boolean"
+          },
+          "optional": {
+            "title": "Is optional and only applies to projects with corresponding npm script",
```

Comment:
>Is optional and only applies to projects with corresponding npm script [](start = 22, length = 70)

Since we're including a title (which I generally don't do in any of my schemas), it should be an English restatement of the variable.

"title": "Ignore Missing Script"